tests/unit_node/http_test.ts

```diff
@@ -314,7 +314,7 @@ Deno.test("[node/http] IncomingRequest socket has remoteAddress + remotePort", a
     remotePort = req.socket.remotePort;
     res.end();
   });
-  server.listen(async () => {
+  server.listen({ host: "0.0.0.0" }, async () => {
```

Comment:
Can we remove the host and update the request instead to make sure ipv6 is working as expected?

Reply:
Sure. In that case we need to update assertion part instead, as `server.listen` (without host specified) listens on ipv6 (`::1`) on unix and ipv4 (`0.0.0.0`) on windows. See https://github.com/denoland/deno/pull/26264/commits/7def448cf0fab5268d38dfd6381db526652bba8e